docs/man/docker-cp.1.md

```diff
@@ -11,8 +12,9 @@ CONTAINER:PATH HOSTPATH
 
 # DESCRIPTION
 Copy files/folders from a container's filesystem to the host
-path. Paths are relative to the root of the filesystem. Files
-can be copied from a running or stopped container.
+path running the command. Paths are relative to the root of
+the filesystem. Files can be copied from a running or stopped
+container.
```

Comment:
How about, change:
Paths are relative to the root of the filesystem.
to:
The path within the container is relative to the root of the filesystem.

I don't think we need to talk about the target spot because that will just do what people expect.  We need to mention what's going on in the container because it might not be clear if its relative to / or relative to WORKDIR, or something else.